### PR TITLE
update(HTML): web/html/global_attributes

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [Глобальні атрибути@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Global_attributes), [сирці Глобальні атрибути@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/global_attributes/index.md)

Нові зміни:
- [mdn/content@e04d8d2](https://github.com/mdn/content/commit/e04d8d2766c468f149445c0bf438d09f9b2d188c)
- [mdn/content@559c464](https://github.com/mdn/content/commit/559c464755abf4ca999a21ce7f3cb1027143f68f)
- [mdn/content@ac28748](https://github.com/mdn/content/commit/ac2874857a3de0be38430e58068597edf0afa2b2)
- [mdn/content@cebb98a](https://github.com/mdn/content/commit/cebb98a8e3a4aa16e3bf5d4fd2ee4612021e6ddd)
- [mdn/content@b3df2fc](https://github.com/mdn/content/commit/b3df2fcf935d7ab2eac9b81fc33838dddfdc9791)
- [mdn/content@fa8a44b](https://github.com/mdn/content/commit/fa8a44b8bff24a4032181c4fd155c459c0dc9161)
- [mdn/content@d3cdafc](https://github.com/mdn/content/commit/d3cdafcdb4d22e5c55771501e7c80451a96aa032)
- [mdn/content@5c30b5e](https://github.com/mdn/content/commit/5c30b5e4e57d1a77ce56b88b3de1171d57562808)
- [mdn/content@7671d18](https://github.com/mdn/content/commit/7671d18ab2d141906af448d5bb125eebc16d3994)
- [mdn/content@ba96f2f](https://github.com/mdn/content/commit/ba96f2f183353872db6d9242c7d2dffe2dbc0c35)
- [mdn/content@4a2b6ca](https://github.com/mdn/content/commit/4a2b6cafbf9bc5b006eedbdf0e9fdf2c8626d5b6)
- [mdn/content@8c0dfad](https://github.com/mdn/content/commit/8c0dfadb9fb1407a7216b287d1bc79ce3a1f99c8)
- [mdn/content@e800a2a](https://github.com/mdn/content/commit/e800a2ad1a8f024c710a829c111fb3b2a9ca167f)
- [mdn/content@658fb87](https://github.com/mdn/content/commit/658fb8714c56051b1de99b864b22587eec323d3a)

### CODE REVIEW

File: files/uk/web/html/global_attributes/index.md [hunk 1]
```diff
@@ -125,6 +123,13 @@ browser-compat: html.global_attributes
     - порожній рядок чи `yes`, що означає, що елемент буде перекладений.
     - `no`, що означає, що елемент не буде перекладений.
 
+- [`virtualkeyboardpolicy`](/uk/docs/Web/HTML/Global_attributes/virtualkeyboardpolicy)
+
+  - : [Перелічений](/uk/docs/Glossary/Enumerated) атрибут, що використовується для контролю поведінки екранної віртуальної клавіатури на пристроях штибу планшетів, мобільних телефонів та інших, на яких апаратної клавіатури може не бути, для елементів, котрі також підтримують атрибут [`contenteditable`](#contenteditable).
```
Comment (AdriandeCita) [inline files/uk/web/html/global_attributes/index.md:128]: Ну, значить будуть "перелічені"

**Review by AdriandeCita — APPROVED** [review]

**Review by AdriandeCita — CHANGES_REQUESTED** [review]
Є свіжі зміни, станом на зараз принаймні три коміти https://github.com/mdn/content/commits/main/files/en-us/web/html/global_attributes/index.md

**Comment by undead404** [comment]
Оновлено

**Comment by undead404** [comment]
Оновлено

**Comment by undead404** [comment]
Зміни з оновлень – відносно суттєві, тому я не просуваю цей PR сильно

**Review by KlashReik — APPROVED** [review]

**Pull request merged**